ifrs17/Test/ImportStorageTest.ipynb

```diff
@@ -590,6 +932,195 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Openings 1",
+                "\n",
+                "\nImport of Scenario Actuals where DataSource contains openings for scenario and best estimate."
```

Comment:
specify if DN is at inception. I guess not..

Reply:
yes, in this case the DN is not at inception year. Specified!